packages/create-waku/tsup.config.ts

```diff
@@ -2,8 +2,10 @@ import { defineConfig } from 'tsup';
 
 export default defineConfig((options) => ({
   entry: ['src/index.ts'],
+  external: [/package\.json/],
   format: ['esm'],
   minify: !options.watch,
+  clean: true,
   /** @see https://github.com/egoist/tsup/issues/927#issuecomment-2354939322 */
   banner: {
     js: `import { createRequire } from 'module'; const require = createRequire(import.meta.url);`,
```

Comment:
You removed `createRequire`, but Is it needed?

Reply:
We could look into moving to inquirer.js from prompts:

https://github.com/SBoudrias/Inquirer.js

It seems like it is more actively maintained, so I'd imagine the esm support is more up to date and we could then remove this banner.

Reply:
Yeah It feels like prompts is not almost maintained. But both of them have similar features. I wonder @dai-shi thoughts :)

Reply:
Previously, I rejected the idea #339 by @himself65 because I thought it was too early.
But now that we have more active contributors, I think the time is right.
I don't think prompts is too bad, but if you guys have a better alternative, and feel like maintaining it, I'm fine of changing the dependency library, or completely re-implementing it.

Reply:
I tried both enquirer and inquirer and each seems to use dynamic requires of their dependencies, so we'd need to keep this banner even if switching.

So, I don't really see a strong motivator for changing off prompts either yet. 

This issue: https://github.com/terkelg/prompts/issues/393 if it get's worked on should be enough to solve any of the current concerns we have now.

Reply:
Yeah, That's enough since create-waku doesn't have the logic of complex CLI yet.
So I don't have enough motivation on changing other deps.